src/action.ts

```diff
@@ -280,27 +279,26 @@ class Action {
       }
     }
 
-    let executionGraphReport: ExecutionGraphReport | undefined = undefined
+    let executionGraphBundle 
 
     if (executionGraph.status === TaskStatus.Succeeded) {
       try {
-        executionGraphReport = await this.vib.getExecutionGraphReport(executionGraphId)
-        core.setOutput("result", executionGraphReport)
+        executionGraphBundle = await this.vib.getExecutionGraphBundle(executionGraphId)
```

Comment:
We want to download the execution graph bundle regardless the final execution graph status.